pkg/async/pool/pool_test.go

```diff
@@ -119,21 +114,47 @@ func (suite) TestPoolGrows(t *testing.T) {
 		}),
 		ResizeEvery: 1 * time.Millisecond,
 	}
-	ng = runtime.NumGoroutine()
-
 	runPool(context.Background(), s)
 
 	defer s.Cancel()
```

Comment:
@pavelsmejkal do I need to cancel here?

Shouldn't it be enough to close the pool -> with current fix it should work and prove that closing the pool really stops the pool

Reply:
i have removed it